web/src/app/alerts/components/AlertDetails.js

```diff
@@ -115,7 +115,9 @@ export default class AlertDetails extends Component {
       <Card className={this.getCardClassName()}>
         <div style={{ display: 'flex' }}>
           <CardContent style={{ flex: 1, paddingBottom: 0 }}>
-            <Typography variant='h5'>Event Log</Typography>
+            <Typography variant='h5' variantMapping={{ h5: 'h3' }}>
```

Comment:
I am not sure that changing the global variant mapping will solve the heading structure. There are times where h6's shouldn't be a heading and times where h5's should be an h2 or h3 and same with h6. 

In the future with a11y testing implemented, it would catch not consecutive heading structure.